io.github.masterbillwoo.rise_of_the_penguins_gb.yml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.masterbillwoo.rise_of_the_penguins_gb
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+
+command: run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+  - --env=ELECTRON_TRASH=gio
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+
+modules:
+  - name: riseofthepenguinsgb
+    buildsystem: simple
+    build-commands:
+      - npm install
+      - npm run make
+      - mkdir -p /app/main
+      - cp -r out/make/linux/x64/riseofthepenguinsgb-linux-x64/* /app/main/
+      - install -Dm755 ../run.sh /app/bin/run.sh
+      - install -Dm644 io.github.masterbillwoo.rise_of_the_penguins_gb.png /app/share/icons/hicolor/256x256/apps/io.github.masterbillwoo.rise_of_the_penguins_gb.png
+      - install -Dm644 io.github.masterbillwoo.rise_of_the_penguins_gb.desktop /app/share/applications/io.github.masterbillwoo.rise_of_the_penguins_gb.desktop
+      - install -Dm644 io.github.masterbillwoo.rise_of_the_penguins_gb.metainfo.xml /app/share/metainfo/io.github.masterbillwoo.rise_of_the_penguins_gb.metainfo.xml
+    sources:
+      - type: git
+        url: https://github.com/masterbillwoo/rise_of_the_penguins_gb.git
+        commit: main
+  - type: script
```

Comment:
incorrect indentation. And because this is yaml that make for an incorrect syntax. It's the build error.